### DISCUSSION TITLE
Why no decision task scheduled to start time out setting?

### DISCUSSION THREAD

**Discussion by zedongh** [uber/cadence#4685] [Q&A] [answered]
Recently, some workflow executed with only a few event, `WorkflowExecutionStart` and `DecisitionTaskScheduled` then got stuck  for a long time. No `DecisitionTaskStart` event comming.

Activity option has `ActivityTaskScheduledToStartTimeout` but workflow no such options.

Because of no `DecisionTaskCompletedEvent`, I cannot use Reset api. 

Is there any method to make workflow get rid of stuck or just reset it?

**Comment by longquanzheng** [comment, marked as answer]
Because there is no value of having such timeout config. If you expect to fail the workflow, the workflow execution timeout will do it. 

We have improved the reset feature to allow reset on DecisionsScheduled event . Please check the reset option.

**Reply by zedongh** [reply]
Thanks for reply.

**Reply by zedongh** [reply]
I'm working on a pretty old version cadence. 😔

**Reply by zedongh** [reply]
Attach related release info:
https://github.com/uber/cadence/releases/tag/v0.18.0

Attach related pull request: 
https://github.com/uber/cadence/pull/3687

Attach related issues:
https://github.com/uber/cadence/issues/2420
